### ISSUE TITLE
Error when using sails-mongo as session adaptor

### ISSUE THREAD

**Issue by s-devaney** [balderdashy/sails-mongo#347]
I get the following error when trying to use sails-mongo as my session adaptor

`error: Could not load Connect session adapter :: sails-mongo
Error from adapter:
TypeError: object is not a function
    at Hook.SessionHook.initialize (/Users/User/Development/sailstest/node_modules/sails/lib/hooks/session/index.js:168:33)
    at Hook.bound [as initialize] (/Users/User/Development/sailstest/node_modules/sails/node_modules/lodash/dist/lodash.js:729:21)
    at /Users/User/Development/sailstest/node_modules/sails/lib/hooks/index.js:75:14
    at /Users/User/Development/sailstest/node_modules/sails/node_modules/async/lib/async.js:451:17
    at /Users/User/Development/sailstest/node_modules/sails/node_modules/async/lib/async.js:441:17
    at _each (/Users/User/Development/sailstest/node_modules/sails/node_modules/async/lib/async.js:46:13)
    at Object.taskComplete (/Users/User/Development/sailstest/node_modules/sails/node_modules/async/lib/async.js:440:13)
    at processImmediate [as _immediateCallback] (timers.js:330:15)
Do you have the Connect session adapter installed in this project?
Try running the following command in your project's root directory:
npm install sails-mongo
(Note: Make sure the version of the Connect adapter you install is compatible with Express 3/Sails v0.10)`

Here is the code I am using in my session.js file:

` adapter: 'sails-mongo',
  host: 'localhost',
  port: 27017,
  db: 'sails',
  collection: 'sessions'`

I definitely have sails-mongo installed & mongod running

**Comment by alexanderzeillinger** [comment]
:+1: Had the same issue 2 days ago.

**Comment by sailsbot** [comment]
Thanks for posting, @s-devaney.  I'm a repo bot-- nice to meet you!

It has been 30 days since there have been any updates or new comments on this page.  If this issue has been resolved, feel free to disregard the rest of this message.  On the other hand, if you are still waiting on a patch, please:

  + review our [contribution guide](https://github.com/balderdashy/sails/blob/master/CONTRIBUTING.md) to make sure this submission meets our criteria (only _verified bugs_ with documented features, please;  no questions, commentary, or bug reports about undocumented features or unofficial plugins)
  + create a new issue with the latest information, including updated version details with error messages, failing tests, and a link back to [the original issue]().  This allows GitHub to automatically create a back-reference for future visitors arriving from search engines.

Thanks so much for your help!

**Comment by tommybananas** [comment]
+1

**Comment by simonmilz** [comment]
+1
